reactor/ip_addr_changed.sls

```diff
@@ -0,0 +1,27 @@
+{%- set api_servers = salt['pillar.get']('roles:kube-master', '') %}
+{%- set sender      = data['id'] %}
+
+{% if sender in api_servers %}
```

Comment:
I think that wouldn't be the right way to solve it: it doesn't make much sense that a local `kubelet` could be talking to a remote API server. And I think the probability of having a working kubelet on the same machine as a dead API server is really low, while the cost of this solution would be that all the `kubelets` in masters _except one_ would be talking to a foreigner API server...

So I think the `/etc/hosts` in the masters should contain just a simple `127.0.0.1 api api.infra.caasp.local` line.

Anyway, we are currently __not__ installing the `kubelet` in the masters...